.circleci/config.yml

```diff
@@ -19,21 +19,6 @@ commands:
           command: |
               git submodule sync
               git submodule update --init --recursive
-      - run:
-          name: Get changed files
-          command: |
-              if [[ $CIRCLE_PULL_REQUEST ]]; then
-                BASE_COMMIT=$(curl -s -H "Authorization: token a61ecb2fc5b72da54431""1b3db3875c96854958a8" \
-                  https://api.github.com/repos/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME/pulls/$CIRCLE_PR_NUMBER \
```

Comment:
If we revoke this token, people will have to merge `master` into their PRs right?

Reply:
@kevinbarabash I was going to, but you're right. Thank you for pointing that out. I'll leave it for the time being.